packages/reporter/src/commands/command-model.ts

```diff
@@ -9,7 +9,7 @@ const LONG_RUNNING_THRESHOLD = 1000
 
 interface RenderProps {
   message?: string
-  indicator?: string
+  indicator?: 'successful' | 'pending' | 'aborted' | 'bad'
```

Comment:
What does the status 'bad' mean?

Reply:
This is set in three places and meant to represent the response status code however it's been pulled into sessions to indicate if a session failed (we intend to update this later on)
- [xhr](https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cy/commands/xhr.ts#L184) - a status code that does not start with 2
- [requests proxy-logging](https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cypress/proxy-logging.ts#L102) - a non-200 status code
- [sessions](https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cy/commands/sessions.ts#L773) - failed on session validation